mailpoet/lib/Cron/Workers/SendingQueue/SendingErrorHandler.php

```diff
@@ -63,6 +69,14 @@ private function processSoftError(MailerError $error, ScheduledTaskEntity $task,
     $queue = $task->getSendingQueue();
 
     if ($queue instanceof SendingQueueEntity) {
+      if ($error->getOperation() === MailerError::OPERATION_DOMAIN_AUTHORIZATION) {
+        $this->loggerFactory->getLogger(LoggerFactory::TOPIC_NEWSLETTERS)->info(
+          'Paused task in sending queue due to sender domain authorization error',
+          ['task_id' => $task->getId()]
```

Comment:
It's probably not even common for people to use multiple sending domains, but for those that do, what do you think about trying to get the actual sender domain or from address in the context? I was thinking it could make debugging easier but I could also see it being redundant if it's clear from other logs or in the UI.

Reply:
> I could also see it being redundant if it's clear from other logs or in the UI.

Thanks for the comment. Tere is another log with the sender domain from the function getDmarcMessage. I would skip it adding it here too, for now, and add it later if we find we need it for debugging.